Fix typos and plurals (#173)
vincenzopalazzo/material-ui-swing: README.md

./gradlew verifyGoogleJavaFormat
```

> If any error are reported please run the following command to try to fix them

Reviewer: ```suggestion
If any error is reported please run the following command to fix them
```

No try here if the syntax it is correct the linter make the job alone :-) (lukely)

```bash
./gradlew googleJavaFormat
```

> p.s: The gradle plugin works with all the JDK versions >= 9 (or better with java byte code version compatible with the version  55.0)

Reviewer: ```suggestion
P.S: The gradle plugin works with all the JDK versions >= 9 (or better with java byte code version compatible with the version  55.0)
```
